solar_installation/demo/website_view.xml

```diff
@@ -0,0 +1,396 @@
+                                                                    <span class="s_website_form_label_content">Requirement</span>
+                                                                </label>
+                                                                <div class="col-sm">
+                                                                    <input type="text" class="form-control s_website_form_input" name="name" required="true" value="" placeholder="" id="o6pgjfglvt5" data-fill-with="undefined"/>
+                                                                </div>
+                                                            </div>
+                                                        </div>
+                                                        <div class="s_website_form_field mb-3 col-12        s_website_form_dnone" data-name="Field">
+                                                            <div class="row s_col_no_resize s_col_no_bgcolor">
+                                                                <label class="col-form-label col-sm-auto s_website_form_label " style="width: 200px">
+                                                                    <span class="s_website_form_label_content"/>
+                                                                </label>
+                                                                <div class="col-sm">
+                                                                    <input type="hidden" class="form-control s_website_form_input" name="team_id" value="1"/>
+                                                                </div>
+                                                            </div>
+                                                        </div>
+                                                        <div class="s_website_form_field mb-3 col-12        s_website_form_dnone" data-name="Field">
+                                                            <div class="row s_col_no_resize s_col_no_bgcolor">
+                                                                <label class="col-form-label col-sm-auto s_website_form_label " style="width: 200px">
+                                                                    <span class="s_website_form_label_content"/>
+                                                                </label>
+                                                                <div class="col-sm">
+                                                                    <input type="hidden" class="form-control s_website_form_input" name="user_id" value="7"/>
+                                                                </div>
+                                                            </div>
+                                                        </div>
+                                                        <div class="mb-0 py-2 col-12 s_website_form_submit" data-name="Submit Button" style="">
```

Comment:
```suggestion
                                                        <div class="mb-0 py-2 col-12 s_website_form_submit" data-name="Submit Button">
```
if there is nothing inside the attribute then we don't need it so please remove all such occurrences.